spec/soft_deletion_spec.rb

```diff
@@ -397,15 +397,15 @@ def undelete!
         forum = category.forums.create!
         category.soft_delete!
 
-        forum.should_not be_deleted # just did an update_all
-        Forum.find(forum.id).should be_deleted
+        expect(forum).not_to be_deleted # just did an update_all
+        expect(Forum.find(forum.id)).to be_deleted
       end
 
       it "custom sql to delete all" do
         category = DDACategory.create!
         forum = category.forums.create!
-        Forum.should_receive(:mark_as_soft_deleted_sql).and_return "fooo"
-        category.forums.should_receive(:update_all).with("fooo")
+        expect(Forum).to receive(:mark_as_soft_deleted_sql).and_return "fooo"
+        expect_any_instance_of(category.forums.class).to receive(:update_all).with("fooo")
```

Comment:
I had to change this because with Rails 5.1, the object that receives this call and the one in this line aren't the same (not the same object_id) to the mock has no impact.